client/src/components/Rating.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+const Rating = () => {
+    return (
+        <div>
+            <div className="game_rating_details">
```

Comment:
I see there are multiple uses of classNames. Are you targeting them within styled-components, or are you using CSS with styled-components?